dadi/lib/handlers/image.js

```diff
@@ -97,6 +97,10 @@ const ImageHandler = function (format, req, {
   this.exifData = {}
   this.isExternalUrl = !pathname.indexOf('http://') || !pathname.indexOf('https://')
 
+  this.noCache = this.parsedUrl.original.query &&
+    Object.keys(this.parsedUrl.original.query).includes('cache') &&
```

Comment:
```suggestion
```

Reply:
I don't think this line is necessary, is it?

Reply:
You're right, it was only if the value was a boolean and it isn't. That being said, I committed this by accident!